skrub/_expressions/_estimator.py

```diff
@@ -737,13 +737,14 @@ def _get_cv_results_table(self, return_metadata=False, detailed=False):
             for k in result_keys[len(metric_names) :]:
                 if k in self.cv_results_:
                     table.insert(table.shape[1], k, self.cv_results_[k])
+        col_score = f"mean_test_{metric_names[0]}"
         table = table.sort_values(
-            f"mean_test_{metric_names[0]}",
+            col_score,
             ascending=False,
             ignore_index=True,
             kind="stable",
         )
-        return (table, metadata) if return_metadata else table
+        return (table, col_score, metadata) if return_metadata else (table, col_score)
```

Comment:
thanks for starting it @Vincent-Maladiere ! do you think it might be easier to just put the col_score in the metadata, eg `metadata["main_score_column"] = "mean_test_accuracy"` or something like that?

Reply:
Why not! How to handle the case when `return_metadata = False` though? 🤔

Reply:
that function is only called by parallel coord which passes `metadata=True` and `results_` and `detailed_results_` with metadata=False but those wouldn't use the col_score either so it shouldn't be a problem. I guess I am saying the sorting column name is metadata about the table in the same way that the list of log-scale column is and callers that need it should pass metadata=True and retrieve it from there.

as this is a private function it is also perfectly fine to remove the metadata=False option and always return the metadata, and callers that don't need it just discard it, as you prefer!